x-pack/plugin/esql/qa/testFixtures/src/main/resources/inlinestats.csv-spec

```diff
@@ -3976,14 +3976,15 @@ required_capability: inline_stats
 required_capability: fix_join_output_merging
 
 FROM languages_lookup_non_unique_key
+| EVAL country = MV_SORT(country)
 | KEEP country, language_name
 | EVAL language_code = null::integer
 | INLINE STATS MAX(language_code) BY language_code
 | SORT country
 | LIMIT 5
 ;
 
-country:text      |language_name:keyword |MAX(language_code):integer |language_code:integer
+country:keyword   |language_name:keyword |MAX(language_code):integer |language_code:integer
```

Comment:
I am curious - why this change is happening in this pull?

Reply:
This could indeed be part of a separate PR, but the basic changes to CsvTestsDataLoader were made to facilitate views testing, and it was small enough that I thought it convenient to include in this PR. I already expect to have 3 or 4 PRs to complete the first functional Views, and hoped not to have even more. But if the changes to CsvTestsDataLoader and these two csv-spec files are cluttering this PR too much, I can move them out to another PR.